packages/lodestar-beacon-state-transition/src/util/blockRoot.ts

```diff
@@ -25,8 +25,8 @@ import {computeStartSlotAtEpoch} from "./epoch";
  * Return the block root at a recent [[slot]].
  */
 export function getBlockRootAtSlot(config: IBeaconConfig, state: BeaconState, slot: Slot): Root {
-  assert(slot < state.slot);
-  assert(state.slot <= slot + config.params.SLOTS_PER_HISTORICAL_ROOT);
+  assert.lt(slot, state.slot, "cannot get future epochs");
+  assert.lte(state.slot, slot + config.params.SLOTS_PER_HISTORICAL_ROOT);
```

Comment:
Could you recommend an assert message here?

Reply:
maybe "cannot get block root from slot more than SLOTS_PER_HISTORICAL_ROOT in the past"